packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Style the self-hosted Sign-In Widget
+---
+
+This section discusses the customization options that you have when you are self-hosting the Sign-In Widget.
+
+> **Note:** You can try all of these customizations yourself using our [Live Widget](https://developer.okta.com/live-widget/).
+
+## Initial sign-in page
```

Comment:
Could this subheading, and the other subheadings on this page change from an h2 to an h3? They seem too big.

Reply:
Or, maybe this could become an h3, and then the sub-topics--logo, custom buttons, links--could become h4s.